kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/configuration/NMDeviceTypeTest.java

```diff
@@ -0,0 +1,189 @@
+package org.eclipse.kura.nm.configuration;
+
+import static org.junit.Assert.assertEquals;
+
+import org.eclipse.kura.KuraException;
+import org.freedesktop.dbus.types.UInt32;
+import org.junit.Test;
+
+public class NMDeviceTypeTest {
+	
+	NMDeviceType type;
+
+    @Test
+    public void shouldReturnCorrectTypeFromUIntZero() throws InterruptedException, KuraException {
```

Comment:
I think [this comment](https://github.com/mattdibi/kura/pull/2/files#r1092851797) applies here too:

```suggestion
    public void conversionWorksForTypeUnknown() throws InterruptedException, KuraException {
```